app/views/jrsClaimSummary.scala.html

```diff
@@ -0,0 +1,102 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.i18n.{I18nSupport, Messages, MessagesApi}
+@import play.api.Play.current
+@import uk.gov.hmrc.tai.config.ApplicationConfig
+@import uk.gov.hmrc.tai.model.JrsClaims
+@import uk.gov.hmrc.urls.Link
+
+@(viewModel: JrsClaims, appConfig: ApplicationConfig)(implicit request: Request[_], messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+
+
+@employerVar = {@if(viewModel.employers.size > 1){employers} else {employer}}
+
+@main(
+    title = messages("check.jrs.claims.title"),
+    gaCustomTitle = None
+){
+
+
+    <div class="grid-row">
+        <div class="grid grid-2-3">
+            <div class="inner-block">
+
+                <p class="flush--bottom">@Link.toInternalPage(
+                    id=Some("backLink"),
+                    value=Some(Messages("check.jrs.back.link")),
+                    url = "#",
+                    cssClasses=Some("link-back js-visible")
+                    ).toHtml</p>
+
+                <h1 class="heading-xlarge">@messages("check.jrs.claims.title")</h1>
+
+
+                <h2 class="heading-section">@messages("check.jrs.from", viewModel.firstClaimDate(appConfig))</h2>
+
+                @for(employers <- viewModel.employers) {
+                    @if(employers.claims.size > 1) {
+                        <p>@messages("check.jrs.employer.made.multiple.claims.for.you", employers.name)</p>
+                        <ul class="list list-bullet">
+                            @for(claims <- employers.claims) {
+                                <li>@claims.yearAndMonth</li>
+                            }
+                        </ul>
+
+                    } else {
+                        <p>@messages("check.jrs.employer.made.single.claims.for.you", employers.name, employers.claims.head.yearAndMonth)</p>
+
+                    }
+                }
+
+                @if(viewModel.employers.size == 1) {
```

Comment:
Move this to the viewModel, e.g. `viewModel.hasSingleEmployment`

Reply:
Done